Muka-is-home/api
# Home page search

### To Test
You can load fixtures with the following commands or if you have your own data that's fine.

check the Postman documentation for API calls.

[https://www.postman.com/speeding-spaceship-78552/workspace/muka/documentation/20353056-159d22bd-50d1-41a7-ac0e-9779ea79e1aa](url)

#### Loading fixtures
You will need to first create two superusers:
```shell
python manage.py createsuperuser
```

After creating two superusers, you can load the fixtures:
```
python manage.py loaddata content_types states tags counties specializations user_types users user_specializations user_counties content_tags contents
```
